source/differenceWith.js

```diff
@@ -23,6 +23,10 @@ import _curry3 from './internal/_curry3';
  *      const l1 = [{a: 1}, {a: 2}, {a: 3}];
  *      const l2 = [{a: 3}, {a: 4}];
  *      R.differenceWith(cmp, l1, l2); //=> [{a: 1}, {a: 2}]
+ *      // Additionally, if the second set is empty or smaller than the first set
```

Comment:
I don't think this has anything to do with the second set being smaller:

```js
R.differenceWith(R.equals, [1, 2, 3, 3, 3], [4, 5, 6, 7, 8, 9]) //=> [1, 2, 3]
```

Reply:
That's correct - the resulting array is deduplicated via the supplied predicate in all cases.